lib/Site.js

```diff
@@ -25,6 +25,7 @@ const Page = require('./Page');
 const CLI_VERSION = require('../package.json').version;
 
 const BOILERPLATE_FOLDER_NAME = '_markbind/boilerplates';
+const HEADING_INDEXING_LEVEL_DEFAULT = 3;
```

Comment:
1. Let's update the test to check that the default heading indexing level is adhered to.
1. We can also modify the inner site to test for custom indexing level (4?).